src/urllib3/contrib/pyopenssl.py

```diff
@@ -81,6 +81,7 @@ class UnsupportedExtension(Exception):
 # Map from urllib3 to PyOpenSSL compatible parameter-values.
 _openssl_versions = {
     util.PROTOCOL_TLS: OpenSSL.SSL.SSLv23_METHOD,
+    PROTOCOL_TLS_CLIENT: OpenSSL.SSL.SSLv23_METHOD,
```

Comment:
nitpick: we now have three different ways of using `PROTOCOL` values for no great reason, can we unify?

Also, should we export `PROTOCOL_TLS_CLIENT` from `urllib3.util`?

Reply:
I actually don't like having `PROTOCOL_TLS` in `urllib3.util`, it's not our API (and technically it's a deprecated API now) so in v2 maybe we should remove it.

Reply:
Sounds good!